style.css

```diff
@@ -784,3 +865,139 @@ span {
     display: none;
   }
 }
+
+.popup {
+  display: grid;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.btn-popup-close {
+  position: absolute;
+  top: 0%;
+  right: 5%;
+  background: none;
+}
+
+.card-popup {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.card-popup > img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+}
```

Comment:
- [ ] Kindly reduce the height of the image as it takes up a lot of height of the webpage compared to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190452240-4f4ffe3e-41de-4571-96d9-6011d4fac3a4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190452172-3ed78896-4b8a-41c1-ba05-969d36037f89.png)

Reply:
👍 @david63011